app/src/main/java/org/xbmc/kore/ui/NowPlayingFragment.java

```diff
@@ -311,18 +321,7 @@ public void onVolumeMuteClicked(View v) {
         action.execute(hostManager.getConnection(), new ApiCallback<Boolean>() {
             @Override
             public void onSuccess(Boolean result) {
-                if (!isAdded()) return;
-                if (result) {
-                    Resources.Theme theme = getActivity().getTheme();
-                    TypedArray styledAttributes = theme.obtainStyledAttributes(new int[] {
-                            R.attr.colorAccent});
-                    volumeMuteButton.setColorFilter(
-                            styledAttributes.getColor(0,
-                                    getActivity().getResources().getColor(R.color.accent_default)));
-                    styledAttributes.recycle();
-                } else {
-                    volumeMuteButton.clearColorFilter();
-                }
+                setVolumeMuteButton(!result);
```

Comment:
I haven't tested this yet, but something looks odd...
Why are you passing `!result` to `setVolumeMuteButton` when the code inside it is exactly the same that was removed from here (which relied on `result`)?